frontend/src/pages/.gitignore

```diff
@@ -0,0 +1 @@
+test.js
```

Comment:
?

Reply:
c'est un fichier test que je laisse sur mon pc pour garder du code qui me servira plus tard, vu qu'on ne peut pas envoyer de code mort. is it ok?

Reply:
Yeah ok, fine by me!